components/style/properties/properties.mako.rs

```diff
@@ -1893,6 +1897,14 @@ pub mod style_structs {
                             self.${longhand.ident} = longhands::${longhand.ident}::computed_value
                                                               ::T(v.into_iter().collect());
                         }
+                    % elif longhand.ident == "display":
+                        /// Set ${longhand.name}.
```

Comment:
Maybe `Set display` directly? And comment on why we set both?

Reply:
Yeah, it should be commented only with `Set display`.

Reply:
Is it okay to explain like this?

```rust
/// Similar to `set_display` in `gecko`, we need to specify
/// `original_display` when setting `display`; thus, we need
/// to special-case this method.
```

Reply:
Maybe just:

```rust
/// We need to keep track of the original display for hypothetical boxes, so we need to special-case this.
```

Reply:
Thanks!